react-app/src/components/form/DateFormField.tsx

```diff
@@ -23,15 +23,23 @@ const DateFormField = (props: DateFieldFormProps) => {
       render={({ field: { onChange, value }, fieldState: { error } }) => {
         return (
           <LocalizationProvider dateAdapter={AdapterDayjs}>
-            <DateField
+            <DatePicker
               disabled={disabled}
-              fullWidth
               onChange={onChange}
               value={value}
               label={label}
               format={'LL'}
-              required={required}
-              slotProps={{ textField: { error: !!error && !!error.message } }}
+              slotProps={{
+                textField: {
+                  error: !!error && !!error.message,
+                  InputProps: {
+                    style: { fontSize: '14px' }, // Font size for input text
+                  },
+                  InputLabelProps: {
+                    style: { fontSize: '12px' }, // Font size for label
+                  },
+                },
+              }}
```

Comment:
```suggestion
              sx={{
                width: '100%'
              }}
              slotProps={{
                textField: {
                  error: !!error && !!error.message,
                  InputProps: {
                    style: { fontSize: '14px' }, // Font size for input text
                  },
                  InputLabelProps: {
                    style: { fontSize: '14px' }, // Font size for label
                  },
                },
              }}
```
The 12px feels really small, but I can see why you would want that. The size of all the other fields though is 16px, and it looks a little strange side-by-side.

Letting the fields expand a bit and bumping up the font size makes it look like this.
![image](https://github.com/user-attachments/assets/0d93ef60-4d4d-4b69-981b-014f56825b60)